docs/source/configuration/slots.md

```diff
@@ -0,0 +1,89 @@
+# Slots
+
+Slots are insertion points in the Volto rendering tree structure.
+This concept is inherited from Plone ClassicUI artifact `Viewlets`.
+
+## Anatomy
+
+Slots are named, and they can contain a list of different slot components.
+Slots components are also named, and they are registered in the configuration registry using a specific API for slots.
+The main trait of a slot component is that its render are controlled by a list of conditions (predicates).
```

Comment:
```suggestion
The main trait of a slot component is that its renderer is controlled by a list of conditions called {term}`predicates`.
```

We should add the definition of `predicate` and `predicates` to the Plone 6 docs' Glossary. Let's steal from Pyramid: https://docs.pylonsproject.org/projects/pyramid/en/latest/glossary.html#term-predicate. What do you think?

Reply:
Yeah! Great! @tiberiuichim and I we were doubting with the name. This is the confirmation that we were right!